### ISSUE TITLE
could you please support for site http://trove.nla.gov.au/

### ISSUE THREAD

**Issue by decemus** [lovasoa/dezoomify#60]
Dear Sir,

Your work is amazing！

I noticed that your work support Deep Zoom. And it seems that the site http://trove.nla.gov.au/ use deep zoom to show pictures, and i catch urls such as http://nla.gov.au/nla.obj-146669907/dzi?tile=12/0_4.jpg, but dezoomify not working.

Thanks

**Comment by decemus** [comment]
Dear Sir,

I notice that you add a "new site support" label to this issue. But when I test following urls, dezoomify still not work, and show me Error: Unable to find a proper dezoomer for the given URL.
http://nla.gov.au/nla.obj-146669907
http://nla.gov.au/nla.obj-146669907/dzi?tile=12
when I test the following url, dezoomify always show me "Preparing tiles load... (0%)"
http://nla.gov.au/nla.obj-146669907/dzi?tile=

Which urls could make dezoomify working normally? Thank you very much

**Comment by lovasoa** [comment]
Yes, the label system allows to sort issues. This issue has a label, but is not closed. No one has worked on it yet. It helps differentiate real issues with the project with simple feature requests. Many people add for new site support. But if a tile format is specific to just one site, it rarely gets added (though if someone makes a pull request to add support for it, I'll accept it).

**Comment by sistasinistra** [comment]
I'm trying to yoink images from NLA too. I can get the image from the above url no problems in Chrome and Firefox on Windows 8 as long as it has the /view on the URL: http://nla.gov.au/nla.obj-146669907/view . Seadragon is the image viewer. 

I have run into problems with this one however: http://nla.gov.au/nla.obj-138737813/view . On Chrome it inevitably crashes (Aw Snap!). On Firefox it only ever renders a portion from the top left, and goes black when I try to Save As... In Chrome I can see that the working memory is not being maxed out, and although the current cache size is being increased as it processes, it is nowhere near the max cache size. There's plenty of disk space, latest version of all browsers and Windows 8. No extensions enabled. Firefox has gfx.max-alloc-size as exactly  2000000000. I tried dezooming in a Safari on a Mac and it was just blank, though it's an older Mac. If I can get hold of a high end machine (mine's middle of the road) I'll try it on there and report back. However that shouldn't be needed I would have thought since diagnostics don't seem to show resources strain.

**Comment by lovasoa** [comment]
Bot-assisted investigation note:

I checked the NLA/Trove URLs from the issue.

The content is tiled Deep Zoom. Current Dezoomify already has NLA-specific support for `/view` URLs by converting them to `/dzi`, and it also handles NLA’s `?tile=` format for tile URLs. The original bare object URL currently returns an anti-bot/interstitial response to automated fetches, so the remaining issue is reliable discovery from that submitted page/input form rather than support for the tile protocol itself.

I labeled this `autodiscovery`. Direct `/dzi` metadata and `/view` URLs are the reliable inputs I found in this pass.

**Comment by AlexanderDeLargeBrotherSir** [comment]
<img width="2560" height="2048" alt="Image" src="https://github.com/user-attachments/assets/9b936bb3-8ec9-4c54-8799-26bd194d74d1" />

http://nla.gov.au/nla.obj-146669907/dzi?tile=12/{{X}}_{{Y}}.jpg

@zhlynn86, if this is resolved please very kindly close this at your earliest convenience.
